akka-discovery/src/main/scala/akka/discovery/ServiceDiscovery.scala

```diff
@@ -8,15 +8,16 @@ import java.net.InetAddress
 import java.util.Optional
 import java.util.concurrent.CompletionStage
 import java.util.concurrent.TimeUnit
-
 import scala.collection.immutable
 import scala.compat.java8.OptionConverters._
 import scala.concurrent.Future
 import scala.concurrent.duration.FiniteDuration
-
 import akka.actor.{ DeadLetterSuppression, NoSerializationVerificationNeeded }
 import akka.util.HashCode
 
+import scala.jdk.DurationConverters._
```

Comment:
shouldn't this be `import akka.util.JavaDurationConverters._` for cross compile?

Reply:
Argh, I never remember which converters to use correctly...